ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/AddGraphicsWithRenderer/AddGraphicsWithRenderer.cpp

```diff
@@ -183,7 +175,7 @@ Geometry AddGraphicsWithRenderer::createHeart()
   partCollection->addPart(heart);
 
   // Construct polygon from part collection
-  PolygonBuilder* heartBuilder = new PolygonBuilder(origin.spatialReference());
+  PolygonBuilder* heartBuilder = new PolygonBuilder(origin.spatialReference(), this);
   heartBuilder->setParts(partCollection);
 
   return heartBuilder->toGeometry();
```

Comment:
it's kind of odd to me that we create this builder on the heap. We set it's parent to be `this` so it will live for the life of the sample but it serves it's only purpose within this scope (e.g. when we return the geometry from it)